test/specs/attribute_spec.txt

```diff
@@ -160,6 +179,14 @@ out("/>\n");
 <input checked="true"/>
 <input checked="false"/>
 <input checked/>
+^^^
+    if ((var_attrContent1) || (("false") == (var_attrValue0))) {
```

Comment:
hmm...nice, but we don't see the actual mapping. i.e. I don't know which code corresponds to which HTL. maybe we should use a JSON like structure to describe the maps after all :-(

Reply:
> That's why I made an initially small commit, so you had time to approve :-)

yes. but I didn't see the big picture :-) my bad.

> Can you show me with an example what JSON structure you have in mind?

```json
[
{s: "<input checked="${true}"/>", m:"if ((var_attrContent1) || (("false") == (var_attrValue0))) {"},
{s: "<input data-sly-attribute.checked="${true}"/>", m:"if ((var_attrValueEscaped_checked3) || (("false") == (var_attrValue_checked2))) {"},
...
]
```

I start to question, if it is useful at all to have the map-specs for every test here. maybe we should have 1 or 2 bing complex examples, and then just have the map tests only for that one; i.e. in a separate json file.